### DISCUSSION TITLE
Error audio my dataset

### DISCUSSION THREAD

**Discussion by tiomaldy** [coqui-ai/TTS#669] [General Q&A]
I got this error
![image](https://user-images.githubusercontent.com/61519589/126120967-5906abd5-0d94-4f85-a511-0469187208b0.png)
I think was a problem with the path but it works fine with other dataset that are in internet with LJ specch this is my config json
`{
    "run_name": "ljspeech",
    "run_description": "finetune 4241 for align with architectural changes",

    "datasets": [
        {
            "name": "ljspeech",
            "path": "/Users/Gustavo/Desktop/ArchivosEntrenados/gustavo",
            "meta_file_train": "metadata.csv",
            "meta_file_val": null
        }
    ],
    "audio":{
        
        "num_mels": 80,          
        "num_freq": 1025,       
        "sample_rate": 16000,
        
        "frame_length_ms": 50,  
        "frame_shift_ms": 12.5, 
        "preemphasis": 0.98,    
        "min_level_db": -100,   
        "ref_level_db": 20,     
        "power": 1.5,           
        "griffin_lim_iters": 60,
        "signal_norm": true,    
        "symmetric_norm": false, 
        "max_norm": 1,          
        "clip_norm": true,      
        "mel_fmin": 0.0,         
        "mel_fmax": 8000.0,        
        "do_trim_silence": true  
    },

    "distributed":{
        "backend": "nccl",
        "url": "tcp:\/\/localhost:54321"
    },

    

    "model": "Tacotron2",   
    "grad_clip": 1,      
    "epochs": 1000,         
    "lr": 0.0001,            
    "lr_decay": false,      
    "warmup_steps": 400,   
    "windowing": false,      
    "memory_size": -1,       
    "attention_norm": "softmax",   
    "prenet_type": "bn",    
    "use_forward_attn": true,    
    "transition_agent": false,    
    "loss_masking": false,       
    "enable_eos_bos_chars": true, 

    "batch_size": 2,       
    "eval_batch_size":4,   
    "r": 1,                 
    "wd": 0.000001,        
    "checkpoint": true,     
    "save_step": 200,      
    "print_step": 100,      
    "tb_model_param_stats": false,     
    "batch_group_size": 8,  

    "run_eval": true,
    "test_delay_epochs": 2,       
    "min_seq_len": 0,       
    "max_seq_len": 150,     
    "output_path": "/Users/Gustavo/Desktop/TTS-main/tacotron2-ouputs",      
    "num_loader_workers": 1,        
    "num_val_loader_workers": 2,    
    "phoneme_cache_path": "/Users/Gustavo/Desktop/TTS-main/phonemecache",  
    "use_phonemes": true,           
    "phoneme_language": "es",     
    "text_cleaner": "phoneme_cleaners"
}
`

And this is my folder with the dataset
![image](https://user-images.githubusercontent.com/61519589/126121131-3e328102-b9c5-4b24-89b1-2660f63740f9.png)

The other folder with other dataset works fine for training but with my own dataset sound likes it can found the wavs files and i don t know why

**Comment by p0p4k** [comment]
Hi, open your file with utf-8-sig encoding.
`with open("transcript_file, "rt", encoding="utf-8-sig") as f:`
